### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bpamos**
minor changes to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,15 +23,14 @@ $ pip install corva-sdk
...
 
-1. `stream` - works with a real-time data
+1. `stream` - works with real-time data
 2. `scheduled` - works with data at defined schedules/intervals
    (e.g. once an hour)
 3. `task` - works with data on-demand
 
-**Note**: it is recommended to use type hints like in examples below, so that editors and tools can give you better
-support.
+**Note**: It is recommended to use type hints like those in the examples below. This allows for better support from editors and tools.
```
Comment (oleksii-symon-corva-ai) [inline README.md:33]: this line is too long (should be <=120 symbols)

**Review by oleksii-symon-corva-ai — APPROVED** [review]

**Comment by oleksii-symon-corva-ai** [comment]
Good, don't forget to use the latest version of master, when creating a new branch. If created with an old master, just pull the changes into master and merge into your branch.

**Comment by oleksii-symon-corva-ai** [comment]
Also use squash and merge, when ready to let the changes into master

File: README.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ $ pip install corva-sdk
...
 
-1. `stream` - works with a real-time data
+1. `stream` - works with real-time data
 2. `scheduled` - works with data at defined schedules/intervals
    (e.g. once an hour)
 3. `task` - works with data on-demand
 
-**Note**: it is recommended to use type hints like in examples below, so that editors and tools can give you better
-support.
+**Note**: It is recommended to use type hints like those in the examples below. <br>
```
Comment (jordanambra) [inline README.md:33]: I don't think we should use <br>s in a markdown file

Comment (bpamos) [reply README.md:33]: ok

**Pull request merged**